test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -4498,9 +4496,11 @@ TEST_P(FailoverTest, UpdateInitialUnavailable) {
   do {
     CheckRpcSendFailure();
   } while (gpr_time_cmp(gpr_now(GPR_CLOCK_REALTIME), deadline) < 0);
-  WaitForBackend(2, false);
+  WaitForBackend(1, false);
   for (size_t i = 0; i < 4; ++i) {
-    if (i == 2) continue;
+    gpr_log(GPR_INFO, "backend %d request count %d", i,
```

Comment:
This logging is not necessary.  If the request count is wrong, the expectation below will log the incorrect value.